src/dmd/dmodule.d

```diff
@@ -677,7 +677,33 @@ extern (C++) final class Module : Package
             FileName.equalsExt(srcfile.toString(), c_ext) &&
             FileName.exists(srcfile.toString()))
         {
-            filename = global.preprocess(srcfile, ifile);  // run C preprocessor
+            /* Look for "importc.h" by searching along import path.
+             * It should be in the same place as "object.d"
+             */
+            const(char)* importc_h;
+
+            foreach (entry; (global.path ? (*global.path)[] : null))
+            {
+                auto f = FileName.combine(entry, "importc.h");
+                if (FileName.exists(f) == 1)
+                {
+                     importc_h = f;
```

Comment:
Where did you clean this? It is going to leak.

Reply:
`FileName.combine()` may or may not allocate memory, so it cannot be free'd.

Reply:
So you mean we may leak? What `FileName.free()` does? It seems bad design if we leak resources.